src/react.ts

```diff
@@ -14,6 +24,38 @@ import type { INTERNAL_Snapshot as Snapshot } from './vanilla'
 
 const { useSyncExternalStore } = useSyncExternalStoreExports
 
+// customized version of affectedToPathList
+// we need to avoid invoking getters
```

Comment:
@dai-shi ahhh, okay, this change (skipping getters in the output of `affectedToPathList`) is why I've been confused.

I've been watching the output of `useAffectedDebugValue` (maybe a little too religiously in retrospect) and was very confused by the lack of getters.

This made me think "valtio is not working" / "doesn't know my getters were invoked", when obviously I could tell it did, b/c the re-renders still worked correctly.

I finally reproduced the behavior in `proxy-compare` and "get it"--getters (and method invocations) are always in the affected map, and are always handled correctly by `isChanged`, it is merely `affectedToPathList` that is leaving them off.

What was the rationale here? I'm sure it makes sense, just curious b/c it tripped me up. Wondering if I can work the rationale/behavior in a doc/"gotchas" update.

Thanks!

Reply:
Without it, it loops infinitely and hangs, AFAIR.